mlr-slurm/README.md

```diff
@@ -0,0 +1,27 @@
+# Execution
+
+Run `r_make()` to execute the whole workflow.
```

Comment:
Could we also link to https://ropenscilabs.github.io/drake-manual/projects.html and especially https://ropenscilabs.github.io/drake-manual/hpc.html#batch-mode-for-long-workflows for suggestions? Users might want to invoke `r_make()` in batch mode, or better yet, inside a SLURM compute node (the latter of which I do plan to cover at https://ropenscilabs.github.io/drake-manual/hpc.html#batch-mode-for-long-workflows at some point).

Reply:
> or better yet, inside a SLURM compute node

Do you mean by this people are allowed to bypass the scheduler on the master node? In our HPC configuration I explicitly forbid this and do not grant the users even access to the computes (only when they have an active job running on a node which was started by the scheduler). I made the experience that people always choose the "easy" way if there is one, which means bypassing the scheduler and jumping to the computes right away. Having this prominently linked in the mlr example would bring me some questions by certain people :smile: 

Also I am not sure if `nohup` is the best tool to suggest users both on HPCs and standalone servers. I used it some years ago and would not recommend it nowadays (instead I recommend `byobu`). 
In any case, I would like to avoid linking to opinionated tools (no matter which) in such an example? But of course this is your decision in the end :)
I just fear that the example might become a bit bloated if we keep adding text.

If you still want to, just add the links without further ado :+1:

Reply:
> Do you mean by this people are allowed to bypass the scheduler on the master node?

I was thinking people could invoke the scheduler the proper way to deploy a single job that calls `r_make()`. Then, that job could submit other jobs (again, invoking the scheduler) via `drake`. I have done this on SGE, and seems to work well. We avoid consuming resources on the login node that way, especially memory, which would not scale well if 100 people submit `drake` workflows with large plans (10000+ targets each).

In any case, I would still prefer to link to those parts of the manual. I know they lack completeness and depth, but it's only a matter of time before I update them. Just need to find a few free moments.

Reply:
> We avoid consuming resources on the login node that way, especially memory, which would not scale well if 100 people submit drake workflows with large plans (10000+ targets each).

That makes sense. I do not even know if its that easy to submit jobs from compute nodes. But that's not important right now.
I still have to "fight" getting people using the new cluster approach because all people were used to "log in, call `Rscript` and be happy". Version control, reproducibility and even seeds are not yet standard everywhere sadly. Most people just take the easy route.

> In any case, I would still prefer to link to those parts of the manual. I know they lack completeness and depth, but it's only a matter of time before I update them. Just need to find a few free moments.

Sure, no problem. The discussion we had here was about sth else than not including these point.